fsharp-backend/src/BackendOnlyStdLib/LibHttpClient1.fs

```diff
@@ -1,3 +1,4 @@
+/// (deprecated) Library functions in the form "HttpClient::xy_v1"
```

Comment:
I'm not sure the "in the form HttpClient::xy" is helpful - all StdLib functions are in that form. Perhaps we could say "v1 functions in the HttpClient module"?